### PR TITLE
plugins: fix CNI configuration typo

### PR DESCRIPTION
**Pull request by aanm**
Fixes: 829fc5730dc1 ("mtu: Introduce device MTU concept")
Signed-off-by: André Martins <andre@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4718)

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

**Comment by tgraf** [comment]
Should we remove `00-cilium-cni.conf` in the directory altogether? I'm not sure it's useful to be able to install the CNI plugin with `make install`?

**Comment by aanm** [comment]
>Should we remove 00-cilium-cni.conf in the directory altogether? I'm not sure it's useful to be able to install the CNI plugin with make install?

No, that's how I install Cilium. Otherwise I wouldn't be able to see:

`
Jul 03 08:18:29 k8s2 kubelet[4647]: W0703 08:18:29.966848    4647 cni.go:126] Error loading CNI config file /etc/cni/net.d/00-cilium-cni.conf: error parsing configuration: invalid character '}' looking for beginning of object key string
`

File: plugins/cilium-cni/cni-install.sh [hunk 1]
```diff
@@ -36,8 +36,7 @@ else
 	cat > ${CNI_CONF_NAME} <<EOF
 {
     "name": "cilium",
-    "type": "cilium-cni",
-    "mtu": ${MTU}
```
Comment (raybejjani) [inline plugins/cilium-cni/cni-install.sh:40]: Is the mtu config a typo or just no longer used?

**Review by raybejjani — APPROVED** [review]

File: plugins/cilium-cni/cni-install.sh (hunk 1 above)
Comment (aanm) [reply plugins/cilium-cni/cni-install.sh:40]: @raybejjani it was removed here https://github.com/cilium/cilium/commit/829fc5730dc1741a07f8781ce6fc74eaeba4edc9#diff-b48f598e63ec7de2b642021d956206ce the trailing "," in the `"type": "cilium-cni",` is the typo

Comment (tgraf) [reply plugins/cilium-cni/cni-install.sh:40]: @raybejjani The mtu is now configured centrally via the cilium-agent so it applies to all plugins with encap overhead accounting taken into account

**Review by tgraf — APPROVED** [review]
Thanks for fixing up my mess @aanm

**Pull request merged**